orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -238,7 +249,10 @@ def _post_process_after_export(exported_model, enable_custom_autograd_function,
 
 def _post_process_enabling_autograd_fallback(exported_model):
     registered_name_mappings = {}
+    skipped_autograd_function_list = ortmodule._defined_from_envvar("SKIPPED_AUTOGRAD_FUNCTIONS", "").split(",")
     for kclass in torch.autograd.Function.__subclasses__():
+        if full_name(kclass) in skipped_autograd_function_list:
```

Comment:
full_name() is did similar thing with line 259 (considering autograd.Function will not be python built-in module). You can keep either of them.